src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/tests/cache_tuning/resource/rca.conf

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Do you need this entire config file? You can also create an rcaConf object with a json string holding the sub-config you are interested in. See [test here](https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/blob/master/src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/configs/CacheActionConfigTest.java#L26-L50)

Reply:
Presently IT takes in a file path to the rca.conf file. I dont think it provides an option to provide a string. Will confirm this with Joydeep